src/main/java/main/view/Project/IssuesAssignServlet.java

```diff
@@ -0,0 +1,46 @@
+package main.view.Project;
+
+import main.Session;
+import main.model.dto.project.TestResultDto;
+import main.model.dto.project.TestRunDto;
+import main.view.BaseServlet;
+import main.view.IPost;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@WebServlet("/issues/assign")
+public class IssuesAssignServlet extends BaseServlet implements IPost {
+
+    @Override
+    public void doPost(HttpServletRequest req, HttpServletResponse resp) {
+        setPostResponseHeaders(resp);
+        setEncoding(resp);
+        try {
+            Session session = createSession(req);
+            Map<String, Integer> results = new HashMap<>();
+            Integer testRunId = getIntegerQueryParameter(req, "testRunId");
+            Integer testResultId = getIntegerQueryParameter(req, "testResultId");
+            if (testRunId != null) {
```

Comment:
what if I specify both testRunId and testResultId?

Reply:
it will use testRunID and ignore the testResultId. Should it be changed?